client/blocks/comments/post-comment-content.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable-next-line scss/at-import-no-partial-leading-underscore */
 @import "calypso/assets/stylesheets/shared/_rendered-blocks.scss";
```

Comment:
does this break without the underscore?

Reply:
It breaks the build. May be our build config is not set up properly to use those local paths?